nixos/jormungandr.nix

```diff
@@ -109,14 +119,6 @@ in {
         '';
       };
 
-      rest.prefix = mkOption {
-        type = types.str;
-        default = "api";
-        description = ''
-          Http prefix of the rest api.
-        '';
-      };
-
       logger.level = mkOption {
         type = types.string;
```

Comment:
May want to change to types.str to avoid silent concatenation if declared in more than one place.